frontend/src/components/Radio/Radio.stories.tsx

```diff
@@ -93,3 +102,80 @@ const TemplateGroup: Story<RadioProps> = (args) => {
 
 export const RadioStates = TemplateGroup.bind({})
 RadioStates.storyName = 'States and themes'
+
+const PlaygroundTemplate: Story = ({
+  name = 'radio',
+  othersInputName = 'others-input',
+  label,
+  isDisabled,
+  isRequired,
+  hasOthers,
+  ...args
+}) => {
+  const options = useMemo(() => ['Option 1', 'Option 2', 'Option 3'], [])
+  const {
+    handleSubmit,
+    formState: { errors },
+    control,
+    trigger,
+  } = useForm()
+  const radioValue: RadioGroupReturn | undefined = useWatch({
+    name,
+    control,
+  })
+  useEffect(() => {
+    // When unchecking others, manually trigger input validation. This is
+    // to ensure that if you select then unselect Others, the form knows
+    // that the text input is now optional.
+    if (hasOthers && !radioValue?.isOthers) {
+      trigger(othersInputName)
+    }
+  }, [hasOthers, radioValue, trigger, othersInputName])
+  const onSubmit = (data: unknown) => {
+    alert(JSON.stringify(data))
+  }
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <FormControl isInvalid={!isEmpty(errors)} mb={6}>
+        <FormLabel isRequired>{label}</FormLabel>
+        <Controller
+          control={control}
+          name={name}
+          render={({ field }) => (
+            <Radio.RadioGroup {...field}>
+              {options.map((o, idx) => (
+                <Radio key={idx} value={o} {...args}>
```

Comment:
i think field.ref may need to be passed into the first radio for react-hook-form to focus on the radio on error. do you want to try that?

Reply:
hmm that seems to break everything, probably because we're passing the same `ref` to both `RadioGroup` and `Radio`

Reply:
by "break everything" I mean it shows the "Answer is required" error even when there is an answer given

Reply:
resolved in https://github.com/opengovsg/FormSG/pull/2628/commits/ee75b15edfebbef1cf8c3f68547a8ffa71b2011a